src/containers/Storage/StorageNodes/StorageNodes.tsx

```diff
@@ -124,23 +132,34 @@ function StorageNodes({data, tableSettings, visibleEntities, onShowAll}: Storage
 
     if (visibleEntities === VisibleEntities.Space) {
         columns = allColumns.filter((col) => col.name !== TableColumnsIds.Missing);
-
-        return (
-            <EmptyFilter
-                title={i18n('empty.out_of_space')}
-                showAll={i18n('show_all')}
-                onShowAll={onShowAll}
-            />
-        );
     }
-    if (visibleEntities === VisibleEntities.Missing) {
-        return (
-            <EmptyFilter
-                title={i18n('empty.degraded')}
-                showAll={i18n('show_all')}
-                onShowAll={onShowAll}
-            />
-        );
+
+    if (!data.length) {
+        let message;
+
+        if (visibleEntities === VisibleEntities.Space) {
```

Comment:
If it is added one more parameter it will be total mess of if-s.
I suggest to discuss with team any universal message. Something like "No nodes with imposed filter combination".

Reply:
[Changed message on filters combination](https://github.com/ydb-platform/ydb-embedded-ui/pull/228/commits/ef3bf63823449697d0b0ee792e59694e99ebfa94). Now it's more clear. 

Did you mean this or you meant introducing single message for all empty results with filters?

Reply:
As for me now it looks good!